peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -13,6 +13,7 @@
   {{ block.super }}
   <meta property="og:title" content="{{ document.title }}" />
   <meta property="og:type" content="book" />
+  <meta property="og:description" content="A {{ document.get_doc_type_display.lower }} made in {{ document.locality }} {{ document.jurisdiction }}." />
```

Comment:
This is hard to get right generically across the whole site. So I think the title is sufficient.
```suggestion
```